modules/mongodb/mongodb.go

```diff
@@ -87,6 +88,10 @@ func WithPassword(password string) testcontainers.CustomizeRequestOption {
 func WithReplicaSet(replSetName string) testcontainers.CustomizeRequestOption {
 	return func(req *testcontainers.GenericContainerRequest) error {
 		req.Cmd = append(req.Cmd, "--replSet", replSetName)
+		req.WaitingFor = wait.ForAll(
+			wait.ForExec(eval("rs.status().ok")),
+			wait.ForListeningPort("27017/tcp"),
+		).WithDeadline(60 * time.Second)
```

Comment:
question: is the intent here to replace the existing or to merge? Also the order seems reversed I would expect the listen check before the exec?
```suggestion
		req.WaitingFor = wait.ForAll(
			req.WaitingFor,
			wait.ForExec(eval("rs.status().ok")),
		).WithDeadline(60 * time.Second)
```

Reply:
That is much smarter! I imagine that the best solution is to extend the default strategies since this should basically be a extra wait strategy that is applied after the other ones. I will update the PR and thanks for the solution! 😁